tide/master/PixelStreamWindowManager.cpp

```diff
@@ -111,9 +111,20 @@ void PixelStreamWindowManager::openWindow( const QString& uri,
 
     const auto type = _isPanel( uri ) ? ContentWindow::PANEL :
                                         ContentWindow::DEFAULT;
-    ContentPtr content = (stream == PixelStreamWindowManager::WEBBROWSER) ?
-                                    ContentFactory::getWebbrowserContent( uri ) :
-                                    ContentFactory::getPixelStreamContent( uri );
+
+    ContentPtr content;
```

Comment:
Not quite... We could add a ContentType::Whiteboard, along with a matching WhiteboardContent class. But there are two use cases where we want a StreamType as a parameter, not an arbitrary ContentType: PixelStreamWindowManager::openWindow() and the new ContentFactory::getPixelStreamContent().